src/js/common/components/form-elements/ErrorableSelect.jsx

```diff
@@ -98,7 +98,7 @@ class ErrorableSelect extends React.Component {
             autoComplete={this.props.autocomplete}
             value={selectedValue}
             onChange={this.handleChange}>
-          <option value="">{this.props.emptyDescription}</option>
+          {this.props.includeBlankOption && <option value="">{this.props.emptyDescription}</option>}
```

Comment:
@crwallace This is a change to a common component but I made the default state the same as it was previously